reactioncommerce/reaction
# ref: add media information to Variants and Options

Resolves https://github.com/reactioncommerce/reaction-next-starterkit/issues/165
Impact: **minor**  
Type: **refactor**

## Issue
The main issue here is inside the `reaction-next-starterkit`, but this PR is laying the groundwork to make the data available that we need inside the starterkit.

Inside the starterkit, we are dealing with an issue where product images haven't been correctly displayed according to the variants they are attached to.

## Solution
This PR adds media to each `variant` and `option` that is saved inside the `Catalog` collection to make it easier to parse and display the media inside the starterkit.

## Breaking changes
None

## Testing
1. Add various media to both variants and options on a products
1. Publish product, which will in turn publish it to the catalog
1. Check the catalog database to see that variant media was correctly added to variant objects
